app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -431,11 +451,11 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
     
           methods.forEach(method => {
             apiResource.addMethod(method, undefined, {
-              authorizationType: apigateway.AuthorizationType.COGNITO,
-              authorizer: {
-                authorizerId: `${authorizer.ref}`
-                //`${theauthorizer.ref}`
-              }
+              authorizationType: apigateway.AuthorizationType.NONE//apigateway.AuthorizationType.COGNITO,
```

Comment:
Why are we changing the Authorization to None? Do we intend to remove the cognito Authentication completely?

Reply:
This has been done temporarily and will be added in before merge to master